libraries/GCS_MAVLink/GCS.h

```diff
@@ -17,7 +17,9 @@
 #include <AP_Mount/AP_Mount.h>
 #include <AP_Avoidance/AP_Avoidance.h>
 #include <AP_HAL/utility/RingBuffer.h>
-
+#if FRSKY_TELEM_ENABLED == ENABLED
+    #include <AP_Frsky_Telem/AP_Frsky_Telem.h>
+#endif
```

Comment:
agreed, this, if anythign, should be included in the cpp file instead of the header